robottelo/ui/base.py

```diff
@@ -686,6 +686,11 @@ def select(self, target, list_value, wait_for_ajax=True, timeout=30,
         if self.element_type(target) == 'select':
             if isinstance(target, tuple):
                 element = self.wait_until_element(target)
+                if element is None:
```

Comment:
if a TimeoutException occur in wait_until_element, it will return a None element.
The purpose of this PR is not to resolve the StaleElementException as it is more complex, but a step in the way to resolve some missing checking on the selenium element, that is passed directly to  execute_script and that prevent us from sending a None element to execute_script in two cases here:
-  menu_click , that via self.scroll_into_view(element)  execute: self.browser.execute_script(           'arguments[0].scrollIntoView(false);', element)
- select execute:  self.browser.execute_script("arguments[0].click();", "arguments[0].click();",  tertiary_element, tertiary_element)

The StaleElementException is not masked here, I'am investigating on how to partially resolve this behavior. As the main problem occur when selecting an org from dashboard that can refresh between finding the element and clicking or selecting in the menu, the same problem occur some time when logging out (we take in to account only successful tests using session that can fail with behavior).